templates/repo/issue/view_content/comments.tmpl

```diff
@@ -495,9 +495,22 @@
 		{{else if eq .Type 25}}
 			<div class="timeline-item event">
 				<span class="badge">{{svg "octicon-git-branch"}}</span>
-				{{template "shared/user/avatarlink" dict "user" .Poster}}
+				{{if .OriginalAuthor}}
+				{{else}}
+					{{template "shared/user/avatarlink" dict "user" .Poster}}
+				{{end}}
 				<span class="text grey muted-links">
-					<a{{if gt .Poster.ID 0}} href="{{.Poster.HomeLink}}"{{end}}>{{.Poster.Name}}</a>
```

Comment:
I made a small drive-by change here and replaced this line with `{{template "shared/user/authorlink" .Poster}}` (line R512), as that's how authors are linked elsewhere in this file.